ligo-virgo-emfollowup-tutorial.ipynb

```diff
@@ -907,6 +908,64 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## 7. Submitting observation coordinates to GraceDB\n",
+    "\n",
+    "Suppose you have performed some EM observations to follow up on a candidate GW event, and you now want to supply the coordinates of those observations to the LV-EM group.  The first step is to obtain a robotic access password from GraceDB and add it to a netrc file (see Section 2). One can then use the Python GraceDB client in order to submit a list of coordinates to GraceDB. To install the GraceDB client package:\n",
+    "\n",
+    "    $ pip install \"ligo-gracedb>=1.18.dev0\" \n",
+    "\n",
+    "Note: It is highly recommended to install the GraceDB client (as well as other packages described used in this tutorial) inside a [virtual environment](https://virtualenv.pypa.io/en/latest/). This isolates the packages required for interacting with GraceDB from other packages on the system.\n",
+    "Now that the GraceDB client is installed, one can use a script such as this to submit observation records to GraceDB:\n"
```

Comment:
Brian tried to write the ligo-gracedb client with requests, but it didn't work for all purposes. (If I remember correctly, the problem had to with auth for file uploads. So it might work for this particular case.) It's worth a try, I guess. But people might get some added value from using the client.